.gitignore

```diff
@@ -61,6 +61,9 @@ BenchmarkDotNet.Artifacts/
 project.lock.json
 project.fragment.lock.json
 artifacts/
+launchSettings.json
+appsettings.Development.json
```

Comment:
Use pattern for this file, but not exact name, because in future we can have different configurations for different environments, e.g. appsettings.Release.json, appsettings.Test.json, appsettings.Production.json